src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const MALE = 'm';
+  const notMarriedMessage = 'I am not married';
```

Comment:
Static constants should be placed outside the component so that they are not created anew after each rendering.